sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -761,6 +762,30 @@ dynamic parameter for the buffer can be registered with all the nodes which
 use the buffer as a parameter. Then a single `dynamic_parameter::update()` call
 will maintain the graphs data dependencies.
 
+===== Whole Graph Update [[whole-graph-update]]
+
+A graph in the executable state can have all of its nodes updated using the
+`command_graph<graph_state::executable>::update(graph)` method. This method
+takes a graph in the modifiable state and updates the nodes in the executable
+state graph to reflect any changes made to the nodes in the modifiable state
+graph. The characteristics which will be updated are detailed in the section on
```

Comment:
I'd suggest using the terms "target graph" and "source graph" whenever you talk about the whole graph update feature.  I think this would be more clear than saying "the modifiable state graph".  I'd also change the name of the formal parameter in the `update` member function to be:

```
void update(const command_graph<graph_state::modifiable>& source);
```

Reply:
Good point, I've updated the wording for this.